scripts/zkai_d128_range_policy_discipline_gate.py

```
@@ -0,0 +1,783 @@
+    expect_equal(source.get("decision"), spec.decision, f"{spec.source_id} decision")
+    if not str(source.get("target_id", "")).startswith("rmsnorm-swiglu-residual"):
+        raise D128RangePolicyError(f"{spec.source_id} target_id drift")
+    if spec.dimension == "d64":
+        expect_equal(source.get("width"), 64, f"{spec.source_id} width")
+    if spec.dimension == "d128":
+        expect_equal(source.get("width"), 128, f"{spec.source_id} width")
+    verifier_domain = source.get("verifier_domain")
+    if not isinstance(verifier_domain, str) or not verifier_domain:
+        raise D128RangePolicyError(f"{spec.source_id} verifier_domain missing")
+    if spec.dimension not in verifier_domain:
+        raise D128RangePolicyError(f"{spec.source_id} verifier domain dimension drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)